src/components/20-molecules/policy-features/policy-features-item/index.js

```diff
@@ -59,6 +58,6 @@ class AXAPolicyFeaturesItem extends LitElement {
   }
 }
 
-defineOnce(AXAPolicyFeaturesItem.tagName, AXAPolicyFeaturesItem);
+defineVersioned([AXAPolicyFeaturesItem], __VERSION_INFO__);
```

Comment:
... and this is standalone versioning, and it will be deduplicated / defined just once in case the version agrees with above. Looks right to me, too.